push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/kafka/MessageHolderWithVariantsKafkaConsumer.java

```diff
@@ -1,44 +0,0 @@
-package org.jboss.aerogear.unifiedpush.message.kafka;
-
```

Comment:
If we remove the producer, how will we handle this https://github.com/aerogear/aerogear-unifiedpush-server/blob/GSOC_2017_kafka/push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/token/TokenLoader.java#L197

Reply:
🤔